packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -6081,8 +6146,25 @@ abstract class RenderObjectElement extends Element {
   RenderObjectElement? _findAncestorRenderObjectElement() {
     Element? ancestor = _parent;
     while (ancestor != null && ancestor is! RenderObjectElement) {
-      ancestor = ancestor._parent;
+      // In debug mode we check whether the ancestor accepts RenderObjects to
+      // produce a better error message in attachRenderObject. In release mode,
+      // we assume only correct trees are built (i.e.
+      // debugAcceptsRenderObjectForSlot always returns true) and don't check
```

Comment:
Is `debugAcceptsRenderObjectForSlot` defined anywhere? I don't see it in the PR, nor in the current code base.

Reply:
Looks like I forgot to update this reference, it has been renamed to debugMustInsertRenderObjectIntoSlot. Will update the comment!

Reply:
I think this was supposed to be `debugMustInsertRenderObjectIntoSlot`